scene/resources/material.cpp

```diff
@@ -988,18 +988,20 @@ void BaseMaterial3D::_update_shader() {
 		case BILLBOARD_DISABLED: {
 		} break;
 		case BILLBOARD_ENABLED: {
-			code += "	MODELVIEW_MATRIX = VIEW_MATRIX * mat4(INV_VIEW_MATRIX[0], INV_VIEW_MATRIX[1], INV_VIEW_MATRIX[2], MODEL_MATRIX[3]);\n";
+			// MAIN_CAM_INV_VIEW_MATRIX is inverse of the camera, even when rendering shadows: this ensures the billboard faces the camera when casting shadows
```

Comment:
```suggestion
			// MAIN_CAM_INV_VIEW_MATRIX is inverse of the camera, even when rendering shadows: this ensures the billboard faces the camera when casting shadows.
```
As per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide), same elsewhere

Reply:
Corrected comment style